### PR TITLE
Format footer

### PR DESCRIPTION
**Pull request by guavafanclub**

### CODE REVIEW

**Comment by guavafanclub** [comment]
This has commits from my last PR, please ignore them. Currently figuring out how to resolve that issue.

**Comment by guavafanclub** [comment]
This has commits from my last PR, please ignore them. Currently figuring out how to resolve that issue.

File: portfolio/src/main/webapp/edu-and-extra.html [hunk 1]
```diff
@@ -52,5 +52,16 @@ <h4>Operations Chair, ColorStack</h4>
...
+        <img src="images/linkedin.png" alt="LinkedIn"></a>
+        <a href="https://github.com/guavafanclub/step">
+        <img src="images/github.png" alt="GitHub"></a>
+      </div>
+      <div id="copyright">
+        <h6>Copyright © 2020 Dee Harris</h6>
+      </div>
+    </div>
   </body>
 </html>
```
Comment (pruess-google) [inline portfolio/src/main/webapp/edu-and-extra.html:67]: Please add a new line at the end of this file

**Comment by alekseychaikovsky** [comment]
Let me know when its ready for review again. Going to leave it for now based on your previous comment

**Pull request closed without merging**